bigquery/params.go

```diff
@@ -492,6 +511,28 @@ func paramValue(v reflect.Value) (*bq.QueryParameterValue, error) {
 	case typeOfIntervalValue:
 		res.Value = IntervalString(v.Interface().(*IntervalValue))
 		return res, nil
+	case typeOfRangeValue:
```

Comment:
Should we add a null value handling for range, together with the rest null values (at row 438)?

Reply:
nevermind, I think it's handled below (row 638) similar to ARRAY and STRUCT.